Unify READMEs (#763)
GetStream/stream-chat-android: stream-chat-android-client/README.md

# Stream Chat Client

> `Stream Chat Client` is the official low-level Android SDK for [Stream Chat](https://github.com/GetStream/stream-chat-android), a service for building chat and messaging applications. This library supports both Kotlin and Java usage. If you can choose we recommend using Kotlin.

Reviewer: Be more opinionated 😄 

```suggestion
`Stream Chat Client` is the official low-level Android SDK for [Stream Chat](https://github.com/GetStream/stream-chat-android), a service for building chat and messaging applications. This library supports both Kotlin and Java usage, but we strongly recommend using Kotlin.
```
